libs/designer/src/lib/core/utils/graph.ts

```diff
@@ -33,11 +34,15 @@ export const DEFAULT_NODE_SIZE = {
   height: 40,
 };
 
+// OpenApi height should be higher than a default node side
+export const OPENAPI_NODE_HEIGHT = 88;
+
 // Creating generic layout nodes and edges below
 
-export const createWorkflowNode = (id: string, type?: WorkflowNodeType): WorkflowNode => ({
+export const createWorkflowNode = (id: string, type?: WorkflowNodeType, operationType?: string): WorkflowNode => ({
   id,
-  ...DEFAULT_NODE_SIZE,
+  width: DEFAULT_NODE_SIZE.width,
+  height: operationType && isOpenApiConnectionType(operationType) ? OPENAPI_NODE_HEIGHT : DEFAULT_NODE_SIZE.height,
```

Comment:
How is OpenApi different from ApiConnection or any other node which has connection, we do not have custom heights for other operations so the issue might be somewhere else, @rllyy97 and @Eric-B-Wu to comment on this.

Reply:
Priti is correct, all of our nodes should have the same default sizing which is just a single-line title and no icon row.  You mentioned it in your issue already, but this is an issue with all nodes getting reset to default sizes in our layout algorithm.  The solution would be to either run the update again or block the reset to defaults from occurring.

I've created an issue for this [here](https://github.com/Azure/LogicAppsUX/issues/2175) to keep track of it.